.github/workflows/ci.yml

```
@@ -13,6 +13,10 @@ env:
   rust_stable: stable
   rust_nightly: nightly-2022-03-21
   rust_clippy: 1.52.0
+  # When updating this, also update:
+  # - README.md
+  # - tokio/README.md
+  # - CONTRIBUTING.md
```

i believe `Cargo.toml`'s `rust-version` keys will also need to be updated (e.g. https://github.com/tokio-rs/tokio/blob/71e18f7b754b29741542bc21bdf99cd30b2fd605/tokio/Cargo.toml#L11)
```suggestion
  # - CONTRIBUTING.md
  # - tokio/Cargo.toml
  # - tokio-util/Cargo.toml
  # - tokio-test/Cargo.toml
  # - tokio-stream/Cargo.toml
```